src/scheduler/main.py

```
@@ -984,3 +992,38 @@ def check_plugin_changes(_type: Literal["external", "pro"] = "external"):
+def kubernetes_start():
+    logger.info(f'{pod_name} is leader')
+    start()
+
+
+def onstopped_leading():
+    logger.info(f'{pod_name} on stop leading, will exit 0')
+    _exit(0)
+
+
+if __name__ == "__main__":
+    SCHEDULER = JobScheduler(environ, logger, INTEGRATION, db=Database(logger, sqlalchemy_string=dotenv_env.get("DATABASE_URI", getenv("DATABASE_URI", None))))  # type: ignore
```

Ruff (0.8.2)

Undefined name `INTEGRATION`

(F821)